iocage/lib/iocage.py

```diff
@@ -391,10 +391,13 @@ def chroot(self, command):
             {
                 "level": "INFO",
                 "message":
-                ("iocage chroot is deprecated. If you need to execute a shell"
-                 " inside the jail use: iocage console" if len(command) == 0
-                 else "iocage chroot is deprecated. If you need to execute a"
-                 " command inside the jail use: iocage exec")
+                (
+                    "iocage is deprecated. "
```

Comment:
`chroot` not iocage :D

Reply:
Of course! :D Fixed.

I will update `lib/ioc_upgrade.py` soon, in few hours.